src/main/java/org/sitenv/ccdaparsing/model/CCDAMedicalEquipment.java

```diff
@@ -0,0 +1,90 @@
+    private CCDACode                            sectionCode;
+    private List<CCDAID>                        ids;
+    private List<CCDAMedicalEquipmentOrg>       equipmentOrgs;
+    private List<CCDANonMedicalSupplyAct>       supplyActivities;
+    private List<CCDAProcActProc>               procedureActs;
+    private ArrayList<String>                   referenceLinks;
+
+    public List<CCDAID> getIds() {
+        return ids == null ? new ArrayList<CCDAID>() : new ArrayList<>(ids);
+    }
+
+    public void setSectionNullFlavourWithNI(boolean isSectionNullFlavourWithNI) {
+        this.isSectionNullFlavourWithNI = isSectionNullFlavourWithNI;
+    }
+
+    public void setSectionTemplateId(ArrayList<CCDAII> templateIds) {
+        this.templateIds = templateIds == null ? new ArrayList<CCDAII>() : new ArrayList<>(templateIds);
+    }
+
+    public void setSectionCode(CCDACode sectionCode) {
+        this.sectionCode = sectionCode;
+    }
+
+    public void setIds(List<CCDAID> ids) {
+        this.ids = ids == null ? new ArrayList<CCDAID>() : new ArrayList<>(ids);
+    }
+
+    public void setProcActsProcs(ArrayList<CCDAProcActProc> procedureActs) {
+        this.procedureActs = procedureActs == null ? new ArrayList<CCDAProcActProc>() : new ArrayList<>(procedureActs);
+    }
+
+    public void setSupplyActivities(List<CCDANonMedicalSupplyAct> supplyActivities) {
+        this.supplyActivities = supplyActivities == null ? new ArrayList<CCDANonMedicalSupplyAct>() : new ArrayList<>(supplyActivities);
+    }
+
+    public void setOrganizers(List<CCDAMedicalEquipmentOrg> equipmentOrgs) {
+        this.equipmentOrgs = equipmentOrgs == null ? new ArrayList<CCDAMedicalEquipmentOrg>() : new ArrayList<>(equipmentOrgs);
+    }
+
+    public boolean isSectionNullFlavourWithNI() {
+        return isSectionNullFlavourWithNI;
+    }
+
+    public ArrayList<CCDAII> getTemplateIds() {
+        return templateIds == null ? new ArrayList<CCDAII>() : new ArrayList<>(templateIds);
+    }
+
+    public CCDACode getSectionCode() {
+        return sectionCode;
+    }
+
+    public List<CCDAMedicalEquipmentOrg> getEquipmentOrgs() {
+        return equipmentOrgs == null ? new ArrayList<CCDAMedicalEquipmentOrg>() : new ArrayList<>(equipmentOrgs);
+    }
+
+    public List<CCDANonMedicalSupplyAct> getSupplyActivities() {
+        return supplyActivities == null ? new ArrayList<CCDANonMedicalSupplyAct>() : new ArrayList<>(supplyActivities);
+    }
+
+    public List<CCDAProcActProc> getProcedureActs() {
+        return procedureActs == null ? new ArrayList<CCDAProcActProc>() : new ArrayList<>(procedureActs);
+    }
+    public ArrayList<String> getReferenceLinks() {
+        return referenceLinks == null ? new ArrayList<String>() : new ArrayList<>(referenceLinks);
+    }
+    public void setReferenceLinks(ArrayList<String> referenceLinks) {
+        this.referenceLinks = referenceLinks == null ? new ArrayList<String>() : new ArrayList<>(referenceLinks);
+    }
+}
```

Comment:
Any particular reason we don't override equals and Code for this class?

Reply:
This was missed out we will add it